projects/storefrontlib/layout/loading/defer-loader.service.ts

```diff
@@ -48,7 +47,7 @@ export class DeferLoaderService {
   }
 
   private shouldLoadInstantly(
-    elementLoadingStrategy: DeferLoadingStrategy
+    elementLoadingStrategy?: DeferLoadingStrategy
```

Comment:
I don't think we should make elementLoadingStrategy optional. Similar reason as [this](https://github.com/SAP/spartacus/pull/15658/files#r870749898)